dvc/repo/artifacts.py

```diff
@@ -38,6 +38,17 @@ def check_name_format(name: str) -> None:
         ) from exc
 
 
+def name_is_compatible(name: str) -> bool:
+    from gto.constants import assert_name_is_valid
```

Comment:
Was it only importing it or using also? If the latter then a noop stub should do. But either way need a comment here with a link so we know this is only kept because of dvclive and can drop it later.

Reply:
> Was it only importing it or using also? 

Actually using